pulp_container/app/downloaders.py

```diff
@@ -51,6 +50,7 @@ async def _run(self, handle_401=True, extra_data=None):
         if extra_data is not None:
             headers = extra_data.get("headers", headers)
             repo_name = extra_data.get("repo_name", None)
+            check_for_existing_manifest = extra_data.get("check_for_existing_manifest", None)
```

Comment:
I'm a bit unhappy with the name. The downloader does not know it downloads a manifest (and i believe it's used for all downloads here). How about `head_only` or `method_head` or even _just_ `method` as a string?

Reply:
+1 to a more generalized name. `method_head` or `head_request` would do for me

Reply:
what about `method` and the default kwargument as `get`?